pkgs/os-specific/darwin/apple-sdk-11.0/frameworks.nix

```diff
@@ -1,7 +1,7 @@
 { frameworks, libs, libobjc, libnetwork }: with frameworks; with libs;
 {
   AGL                              = { inherit Carbon OpenGL; };
-  AVFoundation                     = { inherit ApplicationServices AVFCapture AVFCore CoreGraphics simd UniformTypeIdentifiers; };
+  AVFoundation                     = { inherit ApplicationServices AVFCapture AVFCore CoreGraphics CoreMedia simd UniformTypeIdentifiers; };
```

Comment:
These changes look huge. Should the PR target staging?

Reply:
And I think #204979 does it better.

Reply:
OK, took the approach of https://github.com/NixOS/nixpkgs/pull/204979, and force pushed.